manifests/charts/ai-gateway-helm/templates/deployment.yaml

```diff
@@ -66,8 +74,11 @@ spec:
             periodSeconds: 2
           resources:
             {{- toYaml .Values.controller.resources | nindent 12 }}
-          {{- if .Values.controller.volumes }}
           volumeMounts:
+            - mountPath: /certs
+              name: certs
```

Comment:
In controller-runtime the default webhook certs path is `/tmp/k8s-webhook-server/serving-certs` although we define `tlsCertDir`.

We can use options to override CertDir
```
mgr, err := ctrl.NewManager(cfg, ctrl.Options{
    WebhookServer: webhook.NewServer(webhook.Options{
        Port:     9443,                              // defaults to 9443
        CertDir:  "/etc/webhook/certs",              // override the directory
        CertName: "my-server.crt",                   // override the cert filename
        KeyName:  "my-server.key",                   // override the key filename
    }),
    // …other options…
})
```